random-reviewer-picker/main.js

```diff
@@ -0,0 +1,134 @@
+const github = require("@actions/github");
+const core = require("@actions/core");
+const { random } = require("lodash");
```

Comment:
What about using `Math.random` instead? It would reduce the node_modules by a lot!
Something like:
```js
function getRandomInt(max) {
  return Math.floor(Math.random() * Math.floor(max));
}
```
*Function copied from [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)

Reply:
Actually it wouldn't, I added it because lodash is already used by github

Reply:
Oh, no worries then